Automattic/delphin
# Signup: Validation

This pull request adds validation to the signup form.
#### Testing instructions
1. Run `git checkout add/signup-validation` and start your server
2. Open the [`Sign` page](http://delphin.localhost:1337/signup)
3. Try to submit the form without an email address
4. Assert that you see an error message telling you to provide an email address
5. Try to submit the form with an invalid email address
6. Assert that you see an error message telling you that the email address is invalid
#### Question

Should we disable the submit button if the form is invalid? The form won't be submitted in this case. Do we need a way to remind the user about the invalid fields if they keep trying to submit with invalid values?
#### Additional notes

There are a lot more cases to handle including:
- Errors from the API
- Validation from the API
- The email verification step

I will handle these in a future PR, so that this one can be merged.
#### Reviews
